tests/golden/defaults/prometheus-pushgateway/prometheus-pushgateway/00_namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations: {}
+  labels:
+    name: syn-prometheus-pushgateway
```

Comment:
while we're at it, shouldn't it also contain `SYNMonitoring=main` or so? (or whatever the label's name, I haven't checked it)

Reply:
Not sure yet. The label is not enough for the rules to be picked up. At least on OpenShift 4.

Reply:
OCP4 requires label `openshift.io/cluster-monitoring=true` to allow the OCP4 cluster monitoring to pick up the rules.

Reply:
> OCP4 requires label openshift.io/cluster-monitoring=true

Thanks! Really really hard to find in the docs...